projects/twitter/eval_f1.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+"""Base script for running official ConvAI2 validation eval for f1.
+This uses a the version of the dataset which does not contain candidates.
+Leaderboard scores will be run in the same form but on a hidden test set.
+"""
+
+from parlai.scripts.eval_model import eval_model, setup_args as base_setup_args
+
+def setup_args(parser=None):
+    parser = base_setup_args(parser)
+    parser.set_defaults(
+        task='twitter',
+        datatype='valid',
```

Comment:
can add 'metrics=f1'  now so it doesnt show other metrics which are confusing apparently

Reply:
done!